rest/model/ipam/network.go

```diff
@@ -0,0 +1,53 @@
+package ipam
+
+// AddrStatus is the status of an address.
+type AddrStatus string
+
+// A list of valid address statuses.
+// The default status is StatusPlanned.
+const (
+	StatusPlanned  AddrStatus = "planned"
+	StatusAssigned AddrStatus = "assigned"
+)
+
+// Address wraps an NS1 /ipam/address resource.
+type Address struct {
```

Comment:
this address struct is updated in [PR-8387](https://github.com/nsone/platform/pull/8387), basically it includes extra field `parent_id`